django/http/response.py

```diff
@@ -408,6 +409,16 @@ def content(self, value):
             content = self.make_bytes(value)
         # Create a list of properly encoded bytestrings to support write().
         self._container = [content]
+        self._clear_cached_text()
+
+    @cached_property
+    def text(self):
+        encoding = self.charset or "utf-8"
+        return self.content.decode(encoding)
```

Comment:
```suggestion
        return self.content.decode(self.charset or "utf-8")
```
It's only a personal suggestion: I can't see a real motivation to create the `encoding` variable.

Reply:
After giving it more thought, I realize that what you mentioned makes a lot of sense.